package.json

```diff
@@ -25,6 +25,7 @@
     "@capacitor-mlkit/face-detection": "7.1.0",
     "@capacitor-mlkit/face-mesh-detection": "7.1.0",
     "@capacitor-mlkit/selfie-segmentation": "7.1.0",
+    "@capacitor-mlkit/subject-segmentation": "file:../capacitor-mlkit/packages/subject-segmentation",
```

Comment:
Please install the pre-release, see https://github.com/capawesome-team/capacitor-mlkit/pull/250#issuecomment-2727041531.